TLM/TLM/Manager/Impl/ExtSegmentEndManager.cs

```diff
@@ -98,6 +98,9 @@ public class ExtSegmentEndManager
             return (segmentId * 2) + (startNode ? 0 : 1);
         }
 
+        public ref ExtSegmentEnd GetEnd(ushort segmentId, bool startNode) => ref ExtSegmentEnds[GetIndex(segmentId, startNode)];
```

Comment:
I would move those two new lines above `GetIndex` so will be grouped together nicely.